### ISSUE TITLE
NA Birds 1

### ISSUE THREAD

**Issue by samuelstevens** [samuelstevens/swin-transformer#1]
Dataset: NA Birds
Data: 100%
Model: Best iNat21 (no hierarchy)

Use a pre-trained swin-v2-base model pre-trained on iNat21 with the best validation loss, fine-tune it on NA Birds, then report the accuracy (and preferably attach training logs, link to WandB dashboards, etc).
